frontend/src/components/GddSend/QrCode.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <b-alert show variant="secondary">
+    <!-- eslint-disable-next-line @intlify/vue-i18n/no-v-html -->
     <span class="alert-text" v-html="$t('form.scann_code')"></span>
```

Comment:
`"scann_code": "<strong>QR Code Scanner</strong> - Scan the QR Code of your partner",`
I am pretty sure this will not work, due to the html tag in the lang file. This is something the library prevents by default, which I disabled to not break the whole thing:
```
rules: {
        // TODO: enable
        '@intlify/vue-i18n/no-html-messages': 'off',
},
```